orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/kubernetes/KubernetesContainerFinder.groovy

```diff
@@ -63,6 +63,15 @@ class KubernetesContainerFinder {
 
     containers.forEach { container ->
       if (container.imageDescription.fromContext) {
+	//  If there are parallel Find images stages , then deploymentDetails would be fetched from it. 
```

Comment:
@ajordens  -   Actually  in the current scenario, we are having the single context variable , when we have the parallel Find images  ,   running in two separate threads , the last thread just updates or overwrite the existing result   , so that was the reason to introduce new variable.
 So looking upstream to all parent/ancestor  Find image would only give the  result output saved by the last thread.
 
Can you suggest/guide us  if we can modify the existing context  in a way  that threads should not overwrite the result of find images . 

Regards,
Jatin